### PR TITLE
Social media tweaks

### PR DESCRIPTION
**Pull request by AlbertSuarez**
## Description
This PR is adding the Slack and Telegram join information and fixing FB page link.

## Related Issues
None.

## Some questions
- [x] I read the contributing guidelines
- [ ] I'm adding a new event/footer link
- [ ] I'm editing an existing event/footer link
- [x] I'm adding a new feature/fixing a bug

If you answered No to question 1, go back and read the [instructions](.github/CONTRIBUTING.md) carefully.

## Additional Notes
None.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -45,8 +45,10 @@ email: contact@hackersatupc.org
 
 # Social media + community variables
 twitter: hackersupc
-fb: hackersatupc
+fb: hackersupc
 fb-community: hackersatupc
+slack: http://slack.hackersatupc.org
```
Comment (felixarpa) [inline _config.yml:50]: Minor: Is not related with this PR, but some browsers (like iOS Safari) warns when entering a `http` (not `https`) site. Is hard to change it? @casassg maybe knows 😄

**Review by felixarpa — APPROVED** [review]
LGTM; left a comment, minor.

File: _config.yml (hunk 1 above)
Comment (AlbertSuarez) [reply _config.yml:50]: Looks like the `https` link is not redirecting to the Slack join page. 😕

Comment (felixarpa) [reply _config.yml:50]: Yep, right now it's only `http`, that's why I am saying it should be in another PR (maybe not in a PR but some _config_ changes 🤔), I would accept this PR, create the `https`s and then change the links ☺️

**Pull request merged**